templates/frontend/objects/article_details.tpl

```diff
@@ -125,8 +125,26 @@
 								{if $author->getLocalizedAffiliation() || $author->getLocalizedBiography()}
 									<div class="additional-author-block">
 										<span class="additional-author-name">{$author->getFullName()|escape}</span>
+										{if $author->getData('orcid')}
+											<br/>
+											{if $author->getData('orcidAccessToken')}
+												<a class="orcid-image-url" href="{$author->getData('orcid')|escape}">
+												{if $orcidIcon}
+													{$orcidIcon}
+												{else}
+													<img src="{$baseUrl}/{$orcidImageUrl}">
+												{/if}
+												</a>
+											{/if}
+											<a href="{$author->getData('orcid')|escape}" target="_blank">
+												{$author->getData('orcid')|escape}
+											</a>
```

Comment:
could maybe this be inserted above directly after the orcid icon, so that we do not need two` <a>` elements?

Reply:
And I will see with @ewhanson if both need to be hyperlinked -- in the default theme the icon is not hyperlinked...

Reply:
Erik said that it would be good to hyperling both i.e. it to be as in guidelines: the ORCID id and icon within the same `<a>` element. Would that work here?

Reply:
And we have just found out that now there is a guideline to display another orcid icon for not authenticated orcids, and Erik said we should use that. I will file a new issue for that, but maybe you would like to consider it already here?